### PR TITLE
s390: Handle login for all consoles

### PR DESCRIPTION
**Pull request by Soulofdestiny**

### CODE REVIEW

**Comment by Soulofdestiny** [comment]
Verification runs:
http://opeth.suse.de/tests/5358
http://opeth.suse.de/tests/5359

File: tests/installation/logpackages.pm [hunk 1]
```diff
@@ -22,12 +22,6 @@ use testapi;
 sub run() {
     # the waiting might take long in case of online update repos being
     # initialized before that screen
-    if (get_var('NEW_DESKTOP_SELECTION')) {
-        assert_screen 'before-role-selection', 300;
-    }
-    else {
-        assert_screen 'before-package-selection', 300;
-    }
```
Comment (okurz) [inline tests/installation/logpackages.pm]: conflict resolution fail?

Comment (Soulofdestiny) [inline tests/installation/logpackages.pm]: fixed

**Comment by okurz** [comment]
the code looks good but zkvm still fails in http://opeth.suse.de/tests/5358#step/consoletest_setup/3

File: lib/susedistribution.pm [hunk 2]
```diff
@@ -403,9 +403,10 @@ sub activate_console {
 
     if ($type eq 'console') {
         # different handling for ssh consoles
-        if (check_var('BACKEND', 's390x') || get_var('S390_ZKVM')) {
+        if (check_var('ARCH', 's390x')) {
             # different console-behaviour for s390x
             type_string("su - $user\n") if $user ne 'root';
+            handle_password_prompt;
```
Comment (coolo) [inline lib/susedistribution.pm]: I think the su needs to come _after_ the password_prompt of the root login

Comment (Soulofdestiny) [inline lib/susedistribution.pm]: rerunning right now, will open new PR

Comment (Soulofdestiny) [inline lib/susedistribution.pm]: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3019

**Pull request merged**